fastdds_python/src/swig/fastdds.i

```diff
@@ -365,10 +358,17 @@ namespace builtin {
 %include "fastrtps/Domain.i"
 */
 
-%include "fastdds/dds/core/status/StatusMask.i"
 %include "fastdds/rtps/common/EntityId_t.i"
 %include "fastdds/rtps/common/GuidPrefix_t.i"
 %include "fastdds/rtps/common/Guid.i"
+
+/*
+%include "fastdds/rtps/common/BinaryProperty.i"
+%include "fastdds/rtps/common/Property.i"
+%include "fastdds/rtps/attributes/PropertyPolicy.i"
+*/
```

Comment:
Why are you keeping these commented lines? If they are unused, remove them.

Reply:
Not supported files are right now commented. In the future, we will supporting them and uncommenting them.